AUBURN_SAWMILL option added to PlankRunner

vi3tcn opened the pull request:
AUBURN_SAWMILL option added

- added  new Sawmill Location - AUBURNVALE(new WorldPoint(1395, 3368, 0) 
- editted BankLocation at AUBURNVALE to WorldPoint(1413, 3353, 0) to make it closest to Sawmill
- added NpcID:AUBURN_SAWMILL_OPERATOR to support sawmillOperator at AUBURNVALE

gmason0 committed:
Merge pull request #1265 from chsami/development

Commit:
Update PlankRunnerScript.java

Commit:
Merge pull request #1 from vi3tcn/vi3tcn-patch-1-1

Update PlankRunnerScript.java

Commit:
Update BankLocation.java

Commit:
Update PlankRunnerScript.java

Commit:
Update SawmillLocation.java

gmason0 commented on runelite-client/src/main/java/net/runelite/client/plugins/microbot/plankrunner/PlankRunnerScript.java:
```diff
@@ -132,6 +132,10 @@ public boolean run() {
                         }
 
                         var sawmillOperator = Rs2Npc.getNpc(NpcID.POH_SAWMILL_OPP);
+                        if (sawmillOperator == null) {
```
I think a slightly better approach would be to create a collection of ids & look for all of them @ once.

```java
Set<Integer> sawmillNpcs = Set.of(NpcID.POH_SAWMILL_OPP, NpcID.AUBURN_SAWMILL_OPERATOR);
var sawmillOperator = Rs2Npc.getNpcs(n -> sawmillNpcs.contains(n.getId())).findFirst().orElse(null);
```

Kryaex replied:
Shouldn't we just change `getNpcs(int id) -> getNpcs(int... ids)`. If you want to improve lookup times with sets a lot of other code would need to be updated since afaik they almost all use `anyMatch`

gmason0 replied:
anyMatch would be perfectly fine in this situation, no? We are just looking for either of these npcs?

Kryaex replied:
He just did what you commented guess I'll change Rs2Npc myself if I ever need to use it. And, yeah anyMatch works you just mentioned Set so I didn't know if you were worried about the performance of it for large sets.

gmason0 replied:
I think the performance improvement using a reference type to a primitive type is negligible here - the purpose of using a set was simply to perform a single look up compared to having to look each one up individually by checking if null again

Commit:
PlankRunnerScript

as requested changed :
Set<Integer> sawmillNpcs = Set.of(NpcID.POH_SAWMILL_OPP, NpcID.AUBURN_SAWMILL_OPERATOR);
var sawmillOperator = Rs2Npc.getNpcs(n -> sawmillNpcs.contains(n.getId()))
                             .findFirst()
                             .orElse(null);
